src/test/readMeManipulatorTest.ts

```diff
@@ -133,18 +129,16 @@ describe("@fast ReadmeManipulator.updateVersionTag", () => {
 
     const updated = rm.updateLatestTag(parsed, "package-2018-10");
 
-    chai.expect(updated).to.contain("tag: package-2018-10");
+    expect(updated).toContain("tag: package-2018-10");
   });
 
   it("should correctly update readme", async () => {
     const asa = await getCodeBlocksAndHeadings(parsed.markDown);
-    chai
-      .expect(asa)
-      .to.have.all.keys([
+    expect(Object.keys(asa)).toEqual(expect.arrayContaining([
```

Comment:
`toEqual( .... some template instead of value ...)` sounds a little bit evil. :-)